CHANGELOG-5.2.md

```diff
@@ -8,7 +8,7 @@ To get the diff for a specific change, go to https://github.com/symfony/symfony/
 To get the diff between two versions, go to https://github.com/symfony/symfony/compare/v5.2.0...v5.2.1
 
 * 5.2.0 (2020-11-30)
-
+ * feature #39184 [Console] Separate styling helpers out of SymfonyStyle
```

Comment:
this file is generated. revert all the changes.

Reply:
No problem. I was misled by the pull request template, as it said I must update CHANGELOG manually. Is it outdated?

Reply:
That's about the `CHANGELOG.md` file in the component itself (`src/Symfony/Component/Console/CHANGELOG.md`)